components/automate-ui/src/app/modules/infra-proxy/infra-search-bar/infra-search-bar.component.scss

```diff
@@ -1,55 +1,47 @@
 @import "~styles/variables";
 @import "~styles/mixins";
 
-.search-wrapper {
+form {
+  display: flex;
   width: 100%;
+  flex-direction: row;
```

Comment:
Author note: Thinking of our content in a more flexible manner will allow us to write simpler code that can be viewed properly across multiple screen sizes with fewer CSS.